grpc/dispatcher.go

```diff
@@ -24,6 +25,8 @@ func init() {
 
 const (
 	TimeoutDefault = time.Second
+	// Number of gRPC connection in a pool
+	ConnPoolCount = 5
```

Comment:
How do you decide the default?

Reply:
I use half of the avg number of local-gateway pods, since most of the case in production min replica is 2, the probability the connection to distinct local-gateway pods  more than half the number of pods is higher ( to reduce uneven balance)